src/client/App.js

```diff
@@ -35,6 +36,7 @@ function App() {
           </AuthenticatedRoute>
         </Switch>
       </Router>
+      <SendMessageForm />
```

Comment:
The task was not to apply the message form but just to create it for future use

Reply:
So please remove this

Reply:
Yes I removed.Thank you.